client/components/events/EventViewToggle/EventViewToggle.tsx

```diff
@@ -88,13 +89,13 @@ export const EventViewToggle = ({ filteredEvents }: EventViewProps) => {
 
 interface ActiveViewProps {
   activeView: View,
-  filteredEvents: Array<Event>
+  filteredEvents: FilteredEvents
 }
 
-const ActiveView = ({activeView, filteredEvents}: ActiveViewProps) => {
+const ActiveView = ({activeView, filteredEvents }: ActiveViewProps) => {
   
   const getActiveView = () => {
-    switch(activeView){
+    switch (activeView){
       case View.calendar:
         return <EventCalendarView filteredEvents={filteredEvents}/>
```

Comment:
naming nit: If we do filtering outside of these components, then the components shouldn't care whether the events in their props are filtered.

```suggestion
        return <EventCalendarView events={filteredEvents}/>
```